### PR TITLE
Jordan/2183 fix page network block loading

### LINKED ISSUE #2183: Network blocks table shows header before any blocks are loaded
**Describe the bug**

The headers for the blocks table will be displayed for a couple seconds before the blocks are displayed. 

**To Reproduce**
- Hit refresh on the blocks page
- See the table headers floating without blocks

**Expected behavior**

Show a loading graphic while the blocks are loading

### PR DESCRIPTION
**Pull request by jbibla**
Closes #2183

**Description:**

- show a loading block if there are no blocks loaded yet

screen shot 2019-03-06 at 7 44 47 am

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by fedekunze** [comment]
`Transactions` should start on `0` or have a loader too

File: app/src/renderer/components/network/PageNetwork.vue [hunk 1]
```diff
@@ -40,19 +40,23 @@
...
           <div class="column">
             <dl class="info_dl">
               <dt>Total Delegated {{ bondDenom }}</dt>
               <dd id="bonded_tokens">
-                {{ pool.pool && pool.pool.bonded_tokens ? num.pretty(
-                  num.atoms(pool.pool.bonded_tokens)
-                ) : `n/a` }}
+                {{
+                  pool.pool && pool.pool.bonded_tokens
+                    ? num.pretty(num.atoms(pool.pool.bonded_tokens))
```
Comment (fedekunze) [inline app/src/renderer/components/network/PageNetwork.vue]:
```suggestion
                    ? num.full(num.atoms(pool.pool.bonded_tokens))
```

File: app/src/renderer/components/network/PageNetwork.vue [hunk 2]
```diff
@@ -40,19 +40,23 @@
             <dl class="info_dl">
               <dt>Total Liquid {{ bondDenom }}</dt>
               <dd id="loose_tokens">
-                {{ pool.pool && pool.pool.not_bonded_tokens ? num.pretty(
-                  num.atoms(pool.pool.not_bonded_tokens)
-                ) : `n/a` }}
+                {{
+                  pool.pool && pool.pool.not_bonded_tokens
+                    ? num.pretty(num.atoms(pool.pool.not_bonded_tokens))
```
Comment (fedekunze) [inline app/src/renderer/components/network/PageNetwork.vue]:
```suggestion
                    ? num.full(num.atoms(pool.pool.not_bonded_tokens))
```

**Comment by jbibla** [comment]
> Transactions should start on 0 or have a loader too

you mean on new blocks as they come in?

**Comment by fedekunze** [comment]
> you mean on new blocks as they come in?

yes, when you're first redirected to the Network page

**Comment by jbibla** [comment]
the first thing you'll see is the screenshot above. are you saying when new blocks come in, the transaction count can be delayed?

**Comment by faboweb** [comment]
he means (I think) the transaction count (first row third column) should also either show a loader or a "0"

**Pull request merged**